nixos/modules/services/web-apps/hedgedoc.nix

```diff
@@ -1020,6 +1020,7 @@ in
         ${pkgs.envsubst}/bin/envsubst \
           -o ${cfg.workDir}/config.json \
           -i ${prettyJSON cfg.configuration}
+        mkdir -p ${cfg.configuration.uploadsPath}
```

Comment:
I think a tmpfiles rule would be a better fit here

Reply:
It is not a temp file, it is the target of uploaded files from the web frontend. Maybe if you use s3 or something, but on a default configuration, this is where your data resides.